src/support/steps/ll-css-bg-image.ts

```diff
@@ -58,4 +58,113 @@ Then('I must see the correct style in the head', async function (this: ICustomWo
+            'https://new.rocketlabsqa.ovh/wp-content/rocket-test-data/images/butterfly.avif',
+            'https://e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/file_example_TIFF_1MB.tiff',
+            'https://fastly.picsum.photos/id/976/200/300.jpg?hmac=s1Uz9fgJv32r8elfaIYn7pLpQXps7X9oYNwC5XirhO8',
+            'https://rocketlabsqa.ovh/wp-content/rocket-test-data/images/fixtheissue.jpg',
+            'https://mega.wp-rocket.me/avada/wp-content/rocket-test-data/prague-conference-center-1056491.jpg',
+            'https://e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/butterfly%202.avif',
+            'https://new.rocketlabsqa.ovh/wp-content/rocket-test-data/images/paper%C3%A9quipesTest.jpeg',
+            'https://new.rocketlabsqa.ovh/wp-content/rocket-test-data/images/test_internal2.jpg',
+            'https://e2e.rocketlabsqa.ovh/wp-content/plugins/revslidertest1/public/assets/assets/test_internal3.jpg',
+            'https://e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/test.png',
+            'https://e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/image/test3.gif',
+            'https://e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/test_external1.jpeg',
+            'https://e2e.rocketlabsqa.ovh/test.png',
+            'https://upload.wikimedia.org/wikipedia/commons/1/11/Test-Logo.svg',
+            'https://e2e.rocketlabsqa.ovh/kot%C5%82.png',
+            'https://new.rocketlabsqa.ovh/wp-content/rocket-test-data/styles/assets/images/relative1.jpeg',
+            'https://new.rocketlabsqa.ovh/wp-content/rocket-test-data/styles/assets/images/relative2.jpg',
+            'https://e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/lcp/testsvg.svg',
+            'https://new.rocketlabsqa.ovh//wp-content/rocket-test-data/images/wp-rocket.svg',
+            'https://new.rocketlabsqa.ovh/wp-content/rocket-test-data/images/wp-rocket2.svg',
+            'https://e2e.rocketlabsqa.ovh/new.rocketlabsqa.ovh/wp-content/rocket-test-data/images/test_internal2.jpg',
+            'https://new.rocketlabsqa.ovh/wp-content/rocket-test-data/images/nature.jpeg'
+        ]
+    },
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    ll_bg_css_single_colon: {
+        initialImages: [
+            'https://e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/fabio-sasso-UgpCjt4XLTY-unsplash.jpg',
+            'https://e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/underline.png',
+            'https://e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/maxime-lebrun-6g3Akg708E0-unsplash.jpg'
+        ],
+        lazyLoadedImages: [
+            'https://e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/miguel-luis-6wxFtwSuXHQ-unsplash.jpg'
+        ]
+    },
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    ll_bg_css_double_colon:{
+        initialImages: [
```

Comment:
The template has changed a bit. These should be the correct initial & ll images:
ll_bg_css_double_colon:{
        initialImages: [
            `${WP_BASE_URL}/`+'wp-content/rocket-test-data/images/fabio-sasso-UgpCjt4XLTY-unsplash.jpg',
            `${WP_BASE_URL}/`+'wp-content/rocket-test-data/images/lcp/testjpg4.jpg',
            `${WP_BASE_URL}/`+'wp-content/rocket-test-data/images/underline.png',
            `${WP_BASE_URL}/`+'wp-content/rocket-test-data/images/Przechwytywanie.PNG',
            `${WP_BASE_URL}/`+'wp-content/rocket-test-data/images/maxime-lebrun-6g3Akg708E0-unsplash.jpg',
            `${WP_BASE_URL}/`+'wp-content/rocket-test-data/images/Mapang-test.gif',
            `${WP_BASE_URL}/`+'wp-content/rocket-test-data/images/BigJPGImage_20mbmb.jpg',
        ],
        lazyLoadedImages: [
            `${WP_BASE_URL}/`+'wp-content/rocket-test-data/images/lcp/testavif.avif',
            `${WP_BASE_URL}/`+'wp-content/rocket-test-data/images/img_nature.jpg',
            `${WP_BASE_URL}/`+'wp-content/rocket-test-data/images/istockphoto-1184692500-612x612.webp',
            `${WP_BASE_URL}/`+'wp-content/rocket-test-data/images/lcp/testjpg.jpg',
            `${WP_BASE_URL}/`+'wp-content/rocket-test-data/images/lcp/testjpg3.jpg',
            `${WP_BASE_URL}/`+'wp-content/rocket-test-data/images/lcp/testjpg2.jpg',
            `${WP_BASE_URL}/`+'wp-content/rocket-test-data/images/butterfly.avif',
            `${WP_BASE_URL}/`+'wp-content/rocket-test-data/images/lcp/testwebp.webp',
        ]
    }

Reply:
I guess the other e2e is not updated, I have updated it now and change the command